en-US/firefox/whatsnew_74.lang

```diff
@@ -0,0 +1,67 @@
+## NOTE: Demo page at https://www-demo3.allizom.org/firefox/74.0/whatsnew/all/
+## URL: https://www-dev.allizom.org/%LOCALE%/firefox/74.0/whatsnew/all/
+
+
+# HTML page title
+;What’s new with Firefox - Make it harder for Facebook to track you
+What’s new with Firefox - Make it harder for Facebook to track you
+
+
+;Congrats! You’re using the latest version of Firefox.
+Congrats! You’re using the latest version of Firefox.
+
+
+# Main title
+;It’s okay to like Facebook
+It’s okay to like Facebook
+
+
+# "Facebook Container" is a product name and shouldn't be translated.
+;If you still kinda like Facebook but don’t trust them, then try Firefox’s Facebook Container extension and make it harder for them to track you around the web.
+If you still kinda like Facebook but don’t trust them, then try Firefox’s Facebook Container extension and make it harder for them to track you around the web.
+
+
+# "Facebook Container" is a product name and shouldn't be translated.
+;Get Facebook Container
+Get Facebook Container
+
+
+# "Do it for the 'Gram" is a slang phrase used when people do things in life just so they can take pictures to post on Instagram. Alternative: "It works on Instagram"
+;Do it for the ’Gram
+Do it for the ’Gram
+
+
+# "Facebook Container", "Instagram", "Facebook Messenger" and "Workplace" are proper names that shouldn't be translated.
+;Facebook Container also works on other Facebook owned sites like Instagram, Facebook Messenger and Workplace.
+Facebook Container also works on other Facebook owned sites like Instagram, Facebook Messenger and Workplace.
+
+
+# CTA link
+;Make them unfollow you
+Make them unfollow you
+
+
+;That sneaky little button
```

Comment:
I can't find this string on the demo page

Reply:
It's behind a conditional, you need to load the page for some other locale besides en/fr/de for it to be visible. E.g. https://www-demo3.allizom.org/hu/firefox/74.0/whatsnew/all/